tools/scripts/stm32.mk

```diff
@@ -70,6 +70,8 @@ $(PROJECT_TARGET):
 	$(call print,Creating IDE project)
 	$(call mk_dir, $(BUILD_DIR))
 	$(call mk_dir, $(VSCODE_CFG_DIR))
+	@sed -i 's|TargetToolchain=.*|TargetToolchain=STM32CubeIDE|g' $(HARDWARE) $(HIDE)
```

Comment:
these 2 lines are added by mistake ? maybe a bad rebase ?

Reply:
I missed that, my bad. I'll take these out now